### PR TITLE
Refactoring of save&restore modules for the sake of simplification

### PR DESCRIPTION
**Pull request by georgweiss**

### CODE REVIEW

File: app/save-and-restore/app/src/main/java/org/phoebus/applications/saveandrestore/service/SaveAndRestoreClient.java
Comment (shroffk) [inline app/save-and-restore/app/src/main/java/org/phoebus/applications/saveandrestore/service/SaveAndRestoreClient.java:1]: Can we name the package `o.p.a.saveandrestore.client`

Comment (georgweiss) [reply app/save-and-restore/app/src/main/java/org/phoebus/applications/saveandrestore/service/SaveAndRestoreClient.java:1]: Sure

Comment (georgweiss) [reply app/save-and-restore/app/src/main/java/org/phoebus/applications/saveandrestore/service/SaveAndRestoreClient.java:1]: Done

Comment (georgweiss) [reply app/save-and-restore/app/src/main/java/org/phoebus/applications/saveandrestore/service/SaveAndRestoreClient.java:1]: Updated, done now.

**Pull request merged**